### ISSUE TITLE
Is there any way to create or manipulating SVG Elements "offline" ?

### ISSUE THREAD

**Issue by longfangsong** [svgdotjs/svg.js#899]
In some certain occasions, I'd like to use svg.js to insert a lot of (~3k) complex SVG elements into the DOM and manipulate them .

Since each time we add an element, a reflow will be caused, the web page becomes very slow (~15s to render).

So is there any way to create or manipulating SVG Elements "offline" (i.e. things like `$.detach()`) ?

**Comment by longfangsong** [comment]
I've tried creating `SVG.G` and things like that by hand, and then add them into the doc:

eg.

```typescript
let draw = SVG(document.getElementById("container"));
console.time('timer');
let g = new SVG.G();
for(let i=0;i<1000;++i) {
  g.rect(100,100).x(i);
}
draw.add(g);
console.timeEnd('timer');
```

and it takes ~20ms, which does't have obvious difference with this:

```typescript
let draw = SVG(document.getElementById("container"));
console.time('timer');
let g = draw.group();
for(let i=0;i<1000;++i) {
  g.rect(100,100).x(i);
}
console.timeEnd('timer');
```

**Comment by saivan** [comment]
However, if its taking you almost 10ms to draw an svg from a string directly; you might want to rethink your application, perhaps using svg is not the best choice if you plan to have smooth animations of tens-of-thousands of elements. 

Your bottleneck is most likely going to be the draw time if this applies to you.

**Comment by longfangsong** [comment]
It seems like my problem is caused by layout thrashing.
I didn't expected that calling to `x()` or `y()` with an argument may also call `bbox()`. After read the source of svg.js, I've noticed it now.
@saivan Thank you for your reply. And sorry for bothering.
